### ISSUE TITLE
"Thirst" System

### ISSUE THREAD

**Issue by anIceyGirl** [anIceyGirl/SurvivalOverhaul#16]
From Budgie:
"btw i think we should add a thirst system but not in a way thats like the hunger system this one would be there just if you get to max temperature you can drink to keep the effects of heat stroke away and add the thirst as a effect so you have a timer rather than a extra gui piece"

**Comment by ovfudj** [comment]
Yeah it would get tedious having to manage a whole new bar, but it should give at least a buff that gives temperature resistance when you just drink water or get a bottle of water.
